src/commands/build.yml

```diff
@@ -67,6 +67,19 @@ parameters:
     default: false
     description: >
       Extra output for orb developers
+ 
+  persistence:
+    type: boolean
+    default: false
+    description: >
+      add a folder/files from a previous step
+  
+  attach-location:
```

Comment:
I know why these parameter names were picked, but I'm wondering if there's a better option - maybe `use-workspace` and `attach-at`? On that note, should we include a step for persisting back to the workspace with [`persist-to-workspace`](https://circleci.com/docs/2.0/configuration-reference/#persist_to_workspace)?

Reply:
It has been months since I created this PR and it is not fresh in my mind so forgive me but I don't understand what you mean with `persisting back to workspace`.
Regarding the name change, I don't mind going with `attach-at`. My only worry was with this change requiring us to update all our pipelines configs to support this but we are using a forked version of this orb with all the changes we needed to have to make our pipelines work so  ... :)

Reply:
Totally understand :) 

> but I don't understand what you mean with persisting back to workspace.

Since we're moving files into the job with a workspace, I was wondering it it's worth adding a `persist_to_workspace` command to persist new files back to the workspace. It's not a huge deal if this isn't included, since this is a command and one could write the statement in their own job.

Reply:
Well, I actually don't see a use case in persisting files from building a docker image because you usually just push the image to a repository and use that image but feel free to add this step if you find it useful. ;)

Reply:
I suppose that's true :) We'll go ahead and ignore it for now.

Reply:
Then I assume we are ready to go with this PR except on rebasing it on the latest master.
Will you be able to do it? :)